Corona-Warn-App/src/test/java/de/rki/coronawarnapp/appconfig/mapping/CWAConfigMapperTest.kt

```diff
@@ -62,69 +62,96 @@ class CWAConfigMapperTest : BaseTest() {
     }
 
     @Test
-    fun `feature, device time check is disabled`() {
-        val rawConfig = ApplicationConfigurationAndroid.newBuilder()
-            .setAppFeatures(
-                AppFeatures.newBuilder().apply {
-                    addAppFeatures(
-                        AppFeature.newBuilder().apply {
-                            label = "disable-device-time-check"
-                            value = 1
-                        }.build()
-                    )
-                }
-            )
-            .build()
-        createInstance().map(rawConfig).apply {
-            isDeviceTimeCheckEnabled shouldBe false
-            isUnencryptedCheckInsEnabled shouldBe false
-        }
+    fun `findBoolean() should return true on 1`() {
```

Comment:
Added some tests for `findBoolean()` which also cover the edge-cases when the flag contains an invalid value or an exception is thrown. These cases are already covered, so we don't need tests for these edge cases for every flag that uses `findBoolean()`.